### PR TITLE
Add Examples for File

### PR DESCRIPTION
**Pull request by steveklabnik**
This is pretty basic, but it's nice to have something.

r? @alexcrichton

### CODE REVIEW

File: src/libstd/fs/mod.rs [hunk 1]
```diff
@@ -128,6 +128,18 @@ impl File {
     ///
     /// This function will return an error if `path` does not already exist.
     /// Other errors may also be returned according to `OpenOptions::open`.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::io::prelude::*;
```
Comment (alexcrichton) [inline src/libstd/fs/mod.rs]: I believe the prelude can be omitted for for the examples that don't actually do any reading/writing

**Comment by alexcrichton** [comment]
r=me after removing the preludes which I don't think are necessary. Or r=me if they are!

**Comment by steveklabnik** [comment]
Ahh good call. They're not, they just got put in as part of the copy/paste.

**Pull request merged**